proposals/notary + tuf.adoc

```diff
@@ -0,0 +1,84 @@
+== Notary & TUF Proposal
+
+*Name of project:* Notary & TUF
```

Comment:
Tor uses it for their update system for the TBB (in fact they were the first real users). I also believe that Pythons pip ecosystem also uses TUF. Here's the list that TUF maintains: https://theupdateframework.github.io/#integrations.